recipe/version_setup.patch

```diff
@@ -0,0 +1,14 @@
+Do not import SimpleITK in setup.py when cross-compiling. It will fail due
+to the library being another arch. Instead, get the version from the env var
+set by conda-build.
+--- Wrapping/Python/Packaging/setup.py.in	2022-10-02 18:32:22.000000000 +0200
++++ Wrapping/Python/Packaging/setup.py.in	2022-10-02 18:33:33.000000000 +0200
+@@ -11,7 +11,7 @@
+ import re
+ import os.path as osp
+ 
+-from SimpleITK._version import __version__
++__version__ = os.environ.get('PKG_VERSION', '<unknown>')
```

Comment:
Does this change modify "SimpleITK.__version__"? How does the setup.py version effect the conda build?

Should `${BUILD_DIR}/Wrapping/Python/SimpleITK/_version.py` be written/update after the SimpleITK config/build?

Reply:
Nope, this changes `setup.py.in` only so it doesn't import `SimpleITK` (whose `__init__.py` has the side effect of importing the compiled library. which would fail to load during cross-compilation). I am not touching `_version.py` at all so that should work.

Reply:
Right. That import issue has been addressed in 2.2 https://github.com/SimpleITK/SimpleITK/commit/479eb349bf789beea901b3119d2a4f4084a6f1ef